model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/TransferTests.xtend

```diff
@@ -576,4 +576,210 @@ class TransferTests {
             m | m.assertWarning(JsldslPackage::eINSTANCE.transferFieldDeclaration, JslDslValidator.DUPLICATE_FIELD_MAPPING)
         ]
     }
+    
+    @Test
+    def void testTransferInitializerUnmapped() {
+        '''
+            model Test;
+
+            import judo::types;
+
+            transfer T {
+            	initializer;
+            }
+        '''.parse => [
+            assertError(JsldslPackage::eINSTANCE.transferInitializerDeclaration, JslDslValidator.INVALID_DECLARATION, "Initializer is not allowed in unmapped transfer object.")
+        ]
+    }
+
+    @Test
+    def void testTransferInitializerDuplicated() {
+        '''
+			model Test;
+			
+			import judo::types;
+			
+			entity A {}
+			
+			transfer T(A a) {
+				initializer;
+				initializer;
+			}
+        '''.parse => [
+            assertError(JsldslPackage::eINSTANCE.transferInitializerDeclaration, JslDslValidator.DUPLICATE_INITIALIZER)
```

Comment:
Isn't there a human readable message for duplicate initializer?

Reply:
not needed, it has its own error code